tests/_data/fixtures/Tasks/EchoTask.php

```diff
@@ -1,4 +1,5 @@
 <?php
+namespace Phalcon\Test\Fixtures\Tasks;
```

Comment:
Some PHP files has an empty line after `<?php`:
```php
<?php

namespace Phalcon\Test\Fixtures\Tasks;
```
but  some - not:
```php
<?php
namespace Phalcon\Test\Fixtures\Tasks;
```

Could you please use the same style.